applications/StructuralMechanicsApplication/tests/test_constitutive_law.py

```diff
@@ -625,6 +633,46 @@ def initialize_reference_stress(self, strain_size):
     def get_reference_stress(self, i):
         return self.reference_stress[i]
 
+class DeformationLinearIsotropicDamage(Deformation):
+    def __init__(self):
+        Deformation.__init__(self)
+        self.nr_timesteps = 10
+
+    def set_deformation(self, cl_params, i):
+        self.strain = (i+1)/ self.nr_timesteps * self.initial_strain
+        cl_params.SetStrainVector(self.strain)
+
+class DeformationLinearIsotropicDamage3D(DeformationLinearIsotropicDamage):
```

Comment:
Maybe this class could be derived from `DeformationLinearJ2Plasticity`

Reply:
As a general comment for the CL tests, I'd prefer more independence between them. Maybe we can discuss about it in later